mParticle-Apple-SDK/Include/MPRokt.h

```diff
@@ -85,4 +85,15 @@ typedef NS_ENUM(NSInteger, MPColorMode) {
                   config:(MPRoktConfig * _Nullable)config
                callbacks:(MPRoktEventCallback * _Nullable)roktEventCallback;
 
+/**
+ * Used to report a successful conversion without displaying a placement
+ *
+ * @param placementId Unique identifier for the placement
+ * @param catalogItemId Optional dictionary of attributes to customize the placement
```

Comment:
based on the function signature directly below, it's not optional
```suggestion
 * @param catalogItemId Dictionary of attributes to customize the placement
```